src/table.scss

```diff
@@ -593,6 +593,10 @@ $block: #{$fd-namespace}-table;
     }
   }
 
+  &--no-border {
```

Comment:
please rename it to `--no-outer-border` as user can be confused that this is removing all the borders from the table. Thank you.

Reply:
class renamed to --no-outer-border.